.eslintrc.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
     // possible errors
     'array-callback-return': 'error',
     'consistent-return': 'error',
+    curly: 'error',
```

Comment:
As we discussed while looking at #3033.

Reply:
praise: Thank you! This is a welcome addition.